modules/stats/tradingmodule.py

```diff
@@ -135,20 +163,40 @@ def find_open_trade(self, pair: str) -> Optional[Trade]:
                 return trade
         return None
 
-    def update_value_per_timestamp_tracking(self, trade: Trade, ohlcv: dict) -> None:
+    def update_open_trades_value_per_timestamp(self, trade: Trade, ohlcv: dict) -> None:
+        """
+        Method is used to be able to track the open trades value per timestamp
+        """
         trade_opened_at = datetime.timestamp(trade.opened_at) * 1000
         if trade_opened_at == ohlcv['time']:
-            try:
-                self.open_order_value_per_timestamp[ohlcv['time']] += trade.starting_amount
-            except KeyError:
-                self.open_order_value_per_timestamp[ohlcv['time']] = trade.starting_amount
+            self.open_trades_open_per_timestamp[ohlcv['time']] = \
+                self.open_trades_open_per_timestamp.get(ohlcv['time'], 0) + trade.starting_amount
+            self.open_trades_low_per_timestamp[ohlcv['time']] = \
+                self.open_trades_low_per_timestamp.get(ohlcv['time'], 0) + trade.starting_amount
         else:
-            try:
-                self.open_order_value_per_timestamp[ohlcv['time']] += trade.capital
-            except KeyError:
-                self.open_order_value_per_timestamp[ohlcv['time']] = trade.capital
-
-    def update_budget_per_timestamp_tracking(self, ohlcv: dict) -> None:
+            # When trade.low_value is equal to trade.current in final candle, trade.capital could 
+            # be lower than curr_low_capital due to fee.
+            curr_low_capital = trade.low_value * trade.currency_amount
```

Comment:
```suggestion
            curr_low_capital = trade.low_value * trade.currency_amount
```
Is low value for the lowest value? If so trough would be more meaningfull, e.g.:
curr_low_capital = trade.trough_value * trade.currency_amount
If this also applies for trough capital:
curr_trough_capital = trade.trough_value * trade.currency_amount